packages/rolldown/src/cli/arguments/index.ts

```diff
@@ -37,13 +37,13 @@ export const options = Object.fromEntries(
     if (config && config?.abbreviation) {
       result.short = config?.abbreviation
     }
+    // add `do not` prefix for need reverse options description
+    if (config && config.reverse) {
+      result.description = `do not ${result.description}`
```

Comment:
Use `Disable` instead

Reply:
The `use` and `enable` is hard coded in the description string, Do you mean that I replace the enable with disable instead of add prefix (Do not) ?

https://github.com/rolldown/rolldown/blob/1197c99aa8cd9141ef611d6254024273e3ec0e10/packages/rolldown/src/options/output-options.ts#L78

Reply:
Something like
```suggestion
-     result.description = `do not ${result.description}`
+     if (result.description.startsWith("enable") {
+       result.description.replace("enable", "disable")
+     } else {
+       result.description = `disable ${result.description}`
+     }
```

Reply:
replace description `use external live bindings` with `external live bindings`, then https://github.com/rolldown/rolldown/pull/2517/files#diff-45393b8f7b8fa098c265346c4a08841b717ae192e5e4f011b7329f9d78d63961R42 replace it with `Disable` should enough.